app/models/ske_condition.rb

```diff
@@ -0,0 +1,18 @@
+class SkeCondition < OfferCondition
+  detail :language
+  detail :length
+  detail :reason
+
+  VALID_LANGUAGES = [
+    'French',
+    'Spanish',
+    'German',
+    'ancient languages',
```

Comment:
is this the right capitalisation?

Reply:
I believe so.  Assume because the first three are proper nouns.